modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/info/item/provider/LayoutInfoItemLanguagesProvider.java

```diff
@@ -31,9 +48,106 @@
 		return layout.getAvailableLanguageIds();
 	}
 
+	@Override
+	public Set<Locale> getAvailableLanguages(
```

Comment:
Please move this code in the already existing method. You don't need to pass additional parameters: plid == layout.getPlid() and siteGroupId == layout.getGroupId().

Reply:
thanks for pointing a ti !